api-testing/multicloud-api/src/main/java/com/soda/utils/HeadersName.java

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Can you group together the functionality related to all Http calls?
Why to have a different class?
What are we trying to achieve using a different wrapper?

Reply:
This is repeated questions. Already discuss with @kumarashit

Reply:
https://github.com/Click2cloud-Wadhai can you please put the reasons over here

Reply:
@himanshuvar In this HeadersName.java will be add like this header name `public static final  String X_AMZ_CONTENT_SHA256 = "x-amz-content-sha256";` to avoid hardcore every where in HTTP call.